apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-nav-sidebar-open.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_nav_sidebar_open`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export default () => ( {
+	id: 'wpcom-block-editor-nav-sidebar-open',
+	selector:
+		'.edit-post-header .wpcom-block-editor-nav-sidebar-toggle-sidebar-button__button[aria-expanded="false"]',
+	type: 'click',
+	capture: true,
+	handler: () => tracksRecordEvent( 'wpcom_block_editor_nav_sidebar_open' ),
+} );
```

Comment:
Looking more, I don't think we need to add this for the post-editor.  It is already tracking `calypso_editor_sidebar_open`:

https://github.com/Automattic/wp-calypso/blob/f2af21760128db3bdc9fbf287b09f459c2cb0b8d/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/toggle-sidebar-button/index.tsx#L30

We just don't easily see it in our logs since that comes from editing-toolkit and not wpcom-block-editor, but looking at t.gif under network requests should show it happening.